x/distribution/client/cli/tx.go

```diff
@@ -43,7 +43,7 @@ func GetTxCmd(storeKey string, cdc *amino.Codec) *cobra.Command {
 func GetCmdWithdrawRewards(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "withdraw-rewards",
-		Short: "withdraw rewards for either: all-delegations, a delegation, or a validator",
+		Short: "withdraw rewards for either a delegation or a validator",
```

Comment:
Any reason why we are removing the ability to withdraw all delegations? Wouldn't we need this to minimize the impact of changes to clients?

Reply:
We'll still have it - but instead through a multi-Msg transaction. There was no reason to do this in the first place; it just adds store reads which don't need to be in consensus.

Reply:
#postlaunch the CLI should just have a single withdraw-all-rewards, which builds the multi-msg. Thanks for simplifying the state machine :)

Reply:
We could do that #prelaunch too right? That would significantly simplify the UX...

Reply:
Made an issue - https://github.com/cosmos/cosmos-sdk/issues/3271